src/components/coin-icon/getUrlForTrustIconFallback.js

```diff
@@ -0,0 +1,6 @@
+import { toChecksumAddress } from '../../handlers/web3';
+
```

Comment:
Not a biggie, but just noticed this util function is under the components/coin-icon dir.